tasks/EGPSR.tex

```diff
@@ -1,14 +1,13 @@
 \section{Enhanced General Purpose Service Robot}
 \label{test:egpsr}
-Similar to a smart speaker, the robot is asked to execute arbitrary commands requested by an operator.\\
-\textbf{Note 1:} Due to this task's more open-ended nature, the maximum possible score is higher than the scores of the other \STWO{} tasks.\\
-\textbf{Note 2:} The commands in this task are more complex than in the GPSR task in \SONE.\\
+The robot is asked to find problems in the arena on its own and solve them.
 
-\noindent \textbf{Main Goal:} Execute 3 commands requested by the operator.\\
+
+\noindent \textbf{Main Goal:} Solve 3(5?) problems in the arena.\\
```

Comment:
robot has 10 min for this challenge. So I think 5 might be feasible. You can also not specify the number of problems and just go for as many as possible.

Reply:
I believe assigning a number here encourages teams to experience a sense of fulfillment. The number 5 appears to be suitable.

Reply:
I think I'll just leave it at 3 for now. If teams perform well I think increasing the difficulty of tasks should come first before increasing the amount.